docs/contributing/development.md

```diff
@@ -11,22 +11,101 @@
+manage one yourself and use `pipx` to run tests and build docs with `nox`.
 
-You can see all available sessions with `nox --list`:
+* If you're a Windows user, you'll likely want to set up an environment yourself with `conda`/`mamba`.
+* If you're working in a JupyterHub, you'll likely want to set up an environment yourself with `conda`/`mamba`.
+* If you're using an IDE like VS Code or PyCharm, you'll likely want to set up an environment yourself with `venv` or `conda`/`mamba`.
+* If you're using a plain text editor and don't know how to or want to manage a virtual environment, you'll likely want to start with `pipx`.
+
+=== "`venv`"
+
+    `venv` is a virtual environment manager that's built into Python.
+
+    Create and activate the development environment with:
+
+    ```bash
+    python -m venv .venv
+    source .venv/bin/activate
+    ```
+
+    Then install `earthaccess` into the environment in editable mode with the optional development dependencies:
+
+    ```bash
+    python -m pip install --editable ".[dev,test,docs]"
+    ```
+
+
+=== "`conda`/`mamba`"
+
+    `conda` and `mamba` are open-source package and environment managers that are language and platform agnostic.
+    `mamba` is a newer and faster re-implementation of `conda` -- you can use either `conda` or `mamba`
+    in the commands below. See the installation instructions here (we recommend using Miniforge):
+
+    <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>.
+
+    Once you have `conda`/`mamba` installed, you can create and activate the development environment with:
+
+    ```bash
+    mamba env update -f environment.yml
+    mamba activate earthaccess
+    ```
+
+    This will update (or create if missing) the `earthaccess` environment and active it. The `earthaccess` package will
+    be installed into the environment in editable mode with the optional development dependencies.
+
+    !!! note
+
+        On Sept. 23, 2024, the name of the conda environment changed from `earthaccess-dev` to `earthacess` to align with
+        community best practices. If you have an `earthaccess-dev` environment, we recommend deleting it and creating a new one.
+        From the repository root, you can do that with these commands:
+
+        ```bash
+        mamba env update -f environment.yml
+        mamba activate earthaccess
+        mamba env remove -n earthaccess-dev
+        ```
+
+=== "`pipx`+`nox`"
+
+    `pipx` is a tool to help you install and run end-user applications written in Python and `nox` is Python application
+    that automates testing in multiple Python environments. That means, we can let `pipx` and `nox` manage the `earthaccess`
+    development environment(s) entirely without having to set one up locally. See the `pipx` installation instructions here:
+    <https://github.com/pypa/pipx?tab=readme-ov-file#install-pipx>.
+
+    Once you have `pipx` installed, you can either run `nox` without installing it via:
```

Comment:
```suggestion
    Once you have [installed `pipx` following the official instructions](https://github.com/pypa/pipx?tab=readme-ov-file#install-pipx), you can either run `nox` without installing it via:
```

I prefer to avoid bare links in the docs and instead have labeled links.